docs/docs/segment-spotify.ahk

```diff
@@ -0,0 +1,7 @@
+DetectHiddenWindows, On
```

Comment:
I see I wasn't 100% clear on this, we can move this context to ` docs/docs/segment-spotify.md` in a section that lists the Windows specific settings like `autohotkey_script`.

Reply:
My bad. I did it thiniking the link could be downloaded directly from the doc but it's already the case if I put it directly in the page.